index.html

```diff
@@ -1770,13 +1770,14 @@ <h2>Verification Relationships</h2>
 [[DID-SPEC-REGISTRIES]].
       </p>
       <p>
-The information in a <a>DID document</a> MUST be explicit about the
-<a>verification relationship</a> between the <a>DID subject</a> and the
-<a>verification method</a>.
-<a>Verification methods</a> that are not associated with a particular
-<a>verification relationship</a> MUST NOT be used for that <a>verification
```

Comment:
Hmm, why are we removing this? It's testable via a test that attempts to use a verification method listed in `verificationMethods` to perform a verification where the verification relationship does not appear in the DID Document. For example, using a capability invocation key for the purposes of authentication.

Reply:
Looking at the comment I left on the sheet, it looks like the reason we'd remove it is because this is a statement testing software that relies upon the did document rather than a did document processor. E.g. a statement that would describe how to write a VC verification library.

Doing this seemed odd to me so I left that comment. If you feel it would be beneficial to leave it as normative though, I'm alright with that.

Reply:
@kdenhartog's feedback in the spreadsheet on this line was "Make the wording non-normative but leave in there" so I did that. Sounds like Kyle has changed his mind though? I'll revert the commit.